packages/core-library/system/app/internal/blocks/Hub/Settings/SettingsManagement/steps/MaintenanceMode/EnvironmentSelection.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Property of the NCLEX Power.
+ * Reuse as a whole or in part is prohibited without permission.
+ * Created by the Software Strategy & Development Division
+ */
+import GppMaybeOutlinedIcon from "@mui/icons-material/GppMaybeOutlined";
+import TaskAltOutlinedIcon from "@mui/icons-material/TaskAltOutlined";
```

Comment:
maybe we can used the icon from eva if it is not applicable, we can make this one.
```suggestion
   import { GppMaybeOutlinedIcon as GppOutlineIcon, TaskAltOutlinedIcon as TaskOutlinedIcon} from '@mui/icons-material';
```